rsocket-core/src/main/java/io/rsocket/core/FireAndForgetRequesterMono.java

```diff
@@ -98,21 +103,41 @@ public void subscribe(CoreSubscriber<? super Void> actual) {
       return;
     }
 
+    final RequestInterceptor interceptor = this.requestInterceptor;
+    if (interceptor != null) {
+      interceptor.onStart(streamId, FrameType.REQUEST_FNF, p.sliceMetadata());
```

Comment:
Sometimes (many times?) an interceptor might not care about the metadata. I'm wondering if we can avoid the slice in that case? We could for example pass the request `Payload` and then the interceptor can access the metadata from that if it needs it.

Reply:
We can not do that when it comes to reassembling fragmented payload. Thus, the best is Bytebuf for metadata only. Also, if the fragment is rejected because of leasing, we will never have the full payload

Reply:
I guess we can get rid of metadata, for now, until we see a use case for that

Reply:
revised. Do `p.metadata()` now